### ISSUE TITLE
ModuleNotFoundError: No module named 'django_deno'

### ISSUE THREAD

**Issue by PeterFajner** [Dmitri-Sintsov/djk-sample#7]
I'm getting this error when setting djk-sample up according to the docs:

```
bash-3.2$ python3 -m venv djk-sample
bash-3.2$ cd djk-sample
bash-3.2$ source bin/activate
(djk-sample) bash-3.2$ git clone https://github.com/Dmitri-Sintsov/djk-sample.git
Cloning into 'djk-sample'...
remote: Enumerating objects: 2816, done.
remote: Counting objects: 100% (270/270), done.
remote: Compressing objects: 100% (186/186), done.
remote: Total 2816 (delta 147), reused 179 (delta 74), pack-reused 2546
Receiving objects: 100% (2816/2816), 384.42 KiB | 2.54 MiB/s, done.
Resolving deltas: 100% (1891/1891), done.
(djk-sample) bash-3.2$ cd djk-sample
(djk-sample) bash-3.2$ python3 -m pip install -U -r requirements.txt
<snip pip output>
(djk-sample) bash-3.2$ python3 -m pip install -U -r requirements/bs4.txt
Collecting git+https://github.com/Dmitri-Sintsov/djk-bootstrap4.git@v1.0.0 (from -r requirements/bs4.txt (line 1))
  Cloning https://github.com/Dmitri-Sintsov/djk-bootstrap4.git (to revision v1.0.0) to /private/var/folders/qh/pb_9gg555wgc9rw1bvtxwbm80000gn/T/pip-req-build-5aq5lzbd
  Running command git clone -q https://github.com/Dmitri-Sintsov/djk-bootstrap4.git /private/var/folders/qh/pb_9gg555wgc9rw1bvtxwbm80000gn/T/pip-req-build-5aq5lzbd
  Running command git checkout -q fe24d9afed6ab6c3206e3b21f6cb8010100bd424
Using legacy 'setup.py install' for djk-bootstrap4, since package 'wheel' is not installed.
Installing collected packages: djk-bootstrap4
    Running setup.py install for djk-bootstrap4 ... done
Successfully installed djk-bootstrap4-1.0.0
WARNING: You are using pip version 21.1.1; however, version 21.3.1 is available.
You should consider upgrading via the '/Users/peter/code/djk-sample/bin/python3 -m pip install --upgrade pip' command.
(djk-sample) bash-3.2$ mkdir "$VIRTUAL_ENV/djk-sample/logs/"
(djk-sample) bash-3.2$ mkdir "$VIRTUAL_ENV/djk-sample/fixtures/"
(djk-sample) bash-3.2$ python3 manage.py makemigrations club_app event_app
Traceback (most recent call last):
  File "/Users/peter/code/djk-sample/djk-sample/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/core/management/__init__.py", line 425, in execute_from_command_line
    utility.execute()
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute
    django.setup()
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/config.py", line 223, in create
    import_module(entry)
  File "/Users/peter/opt/miniconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'django_deno'
```

Looks like django_deno is [this](https://github.com/Dmitri-Sintsov/django-deno), but it's not in any of the requirements files.

I tried to remove it from INSTALLED_APPS and got this error instead, which seems like a rabbit hole of issues:

```
(djk-sample) bash-3.2$ python3 manage.py makemigrations club_app event_app
Traceback (most recent call last):
  File "/Users/peter/code/djk-sample/djk-sample/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/core/management/__init__.py", line 425, in execute_from_command_line
    utility.execute()
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute
    django.setup()
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/config.py", line 123, in create
    mod = import_module(mod_path)
  File "/Users/peter/opt/miniconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django_jinja_knockout/apps.py", line 6, in <module>
    from . import middleware
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django_jinja_knockout/middleware.py", line 10, in <module>
    from .tpl import format_html_attrs
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django_jinja_knockout/tpl.py", line 34, in <module>
    from .models import model_fields_verbose_names
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django_jinja_knockout/models.py", line 6, in <module>
    from django.contrib.auth.models import User, Permission
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 47, in <module>
    class AbstractBaseUser(models.Model):
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/db/models/base.py", line 108, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/registry.py", line 253, in get_containing_app_config
    self.check_apps_ready()
  File "/Users/peter/code/djk-sample/lib/python3.9/site-packages/django/apps/registry.py", line 136, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

**Comment by Dmitri-Sintsov** [comment]
Old version from pypi is incompatible to newer Django versions.

I should release 2.0 at pypi but there is a lot of unfinished works, for example the docs are not updated yet.

django_deno is required only in case v2.0 is targeted to run in outdated IE11. Otherwise, it may be skipped (or unused).

Please try to use the development version:

In the virtual environment run the following commands:

```
python3 -m pip install -r requirements/dev.txt
$VIRTUAL_ENV/djk-sample/cli/4bs.sh
```
Let me know whether it worked for you.

**Comment by Dmitri-Sintsov** [comment]
Just released v2.0.0 with updated installation instructions for stable / dev versions:

https://pypi.org/project/django-jinja-knockout/2.0.0/
